tbtadm/controller.cpp

```diff
@@ -302,9 +322,16 @@ void tbtadm::Controller::devices()
 
         // TODO: better formatting
         const auto routeString = dir.path().filename().string();
+        if (authorized)
+            m_out << highlight(green);
+
         m_out << routeString << '\t' << readVendor(vendorFilename) << '\t'
-              << readDevice(deviceFilename) << '\t' << authorized() << '\t'
+              << readDevice(deviceFilename) << '\t'
+              << (authorized ? "authorized" : "non-authorized") << '\t'
               << inACL() << '\n';
+
+        if (authorized)
+            m_out << highlight(normal);
```

Comment:
Generally, it's better to use something like "scope guard" to make sure it's restored even if something interrupted the previous actions.
Here, the action itself isn't noexcept so maybe it's a bit more complicate case.
Again, your call.